src/Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY . $WORKDIR
 # installing dependencies to build package
 RUN pip install . -t python
 
-# change 31860
```

Comment:
No, really just scoped this PR for GTC-2390 issue of fire alerts updates and considered it won't be a big deal to add start date manually since the tables are not created frequently.
What you mention and adding the metadata for all tables occurred to me but need to dig more into the code.